README.md

```diff
@@ -135,7 +135,7 @@ completed without error (whether or not changes were made).
 * [Emacs](https://github.com/rust-lang/rust-mode)
 * [Sublime Text 3](https://packagecontrol.io/packages/RustFmt)
 * [Atom](atom.md)
-* Visual Studio Code using [vscode-rust](https://github.com/editor-rs/vscode-rust), [vsc-rustfmt](https://github.com/Connorcpu/vsc-rustfmt) or [rls_vscode](https://github.com/jonathandturner/rls_vscode) through RLS.
+* Visual Studio Code using [rust-analyzer](https://marketplace.visualstudio.com/items?itemName=rust-lang.rust-analyzer) and [vsc-rustfmt](https://github.com/Connorcpu/vsc-rustfmt)
```

Comment:
The link to [`vsc-rustfmt`](https://github.com/Connorcpu/vsc-rustfmt) also looks like a project that isn't being maintained. Maybe remove it and only link to the rust-analyzer VS Code extension?

<img width="1452" alt="Screen Shot 2022-11-05 at 1 39 56 PM" src="https://user-images.githubusercontent.com/29028348/200133680-631f0871-e7a1-445d-9b35-2bacf0681386.png">